classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+  attr_accessor :label
+  attr_reader :students
+
+  def add_student(student)
+    student.classroom = self if student.classroom != self
+    students.push(student) unless students.include?(student)
+  end
```

Comment:
- [ ] This should be `@student.push(student) unless students.include?(student)`. Remember that it is the instance variable `@students` that is an array and a `Student` object will be pushed into this array. 👍